Pods/Pods.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,2660 @@
+// !$*UTF8*$!
+{
```

Comment:
@pushpinderpalsingh i don't think so this file will be pushed at all

Reply:
![image](https://user-images.githubusercontent.com/47032662/88574200-90523a80-d05f-11ea-9d2f-ac602cb8f03f.png)
the pods folder is not ignored in the developed branch. Can you have a look?